requirements-common.txt

```diff
@@ -38,3 +38,4 @@ depyf==0.18.0 # required for profiling and debugging with compilation config
 cloudpickle # allows pickling lambda functions in model_executor/models/registry.py
 watchfiles # required for http server to monitor the updates of TLS files
 python-json-logger # Used by logging as per examples/other/logging_configuration.md
+loguru==0.7.3 # Required for JSON logging
```

Comment:
please help moving this as optional dependencies so people can install `vllm[logging]` instead. our default deps are getting too bloated.